### PR TITLE
Nep 129 create mutation api

### PR DESCRIPTION
**Pull request by shrijan-siris**
- db with user-schema tag is used
- mutation for creating user with fcbtoken and countrycode is made

### CODE REVIEW

File: src/database/resolvers.js [hunk 1]
```diff
@@ -36,8 +38,21 @@ exports.resolver = {
...
-			//   const sortedTweets = getSortedTweets(tweets);
 			return tweets
 		},
 	},
+	Mutation: {
+		storeFcmToken: async (parent, args) => {
+			const {
+				input: { fcmToken, countryCode },
+			} = args
+			console.log('_______________fcmtoken_______________', fcmToken)
```
Comment (syuraj) [inline src/database/resolvers.js]: @shrijan-siris you planning to remove `console.log` from final pr? We should start using `winston` for logging.

Comment (shrijan-siris) [reply src/database/resolvers.js]: not for now we may remove after using winston

Comment (syuraj) [reply src/database/resolvers.js]: yeah, with `winston`, we should be able to enable console logging only during development and also have different log levels like debug, info, warn per environment (dev, prod)

File: src/utils/arryUtil.test.js [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
-			{ source: 'Setopati', weight: 70 },
-			{ source: 'Hariyopati', weight: 70 },
-		]
-		const expected = [
-			{ source: 'Setopati', weight: 70 },
-			{ source: 'Hariyopati', weight: 70 },
-			{ source: 'Ratopati', weight: 60 },
-			{ source: 'Kantipur', weight: 50 },
-		]
+describe("Array Util Test", () => {
```
Comment (syuraj) [inline src/utils/arryUtil.test.js]: @shrijan-siris how did it switched back to double quotes?

File: src/database/resolvers.js [hunk 3]
```diff
@@ -1,48 +1,71 @@
...
+
+      return tweets;
+    }
+  },
+  Mutation: {
+    storeFcmToken: async (parent, args) => {
+      const {
+        input: { fcmToken, countryCode }
+      } = args;
+      console.log("_______________fcmtoken_______________", fcmToken);
```
Comment (syuraj) [inline src/database/resolvers.js]: @shrijan-siris let's remove console.logs from final code. Also device token is a secret information, we probably don't want in logs

**Pull request merged**